vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -399,9 +399,34 @@ - (void)publisherActivityFromURL:(NSURL *)URL
   ledger->GetPublisherActivityFromUrl(1, visitData.Clone(), blob);
 }
 
-- (void)updatePublisherExclusionState:(NSString *)publisherId state:(BATPublisherExclude)state
+- (void)mediaPublisherInfoForMediaKey:(NSString *)mediaKey completion:(void (^)(BATPublisherInfo * _Nullable))completion
```

Comment:
looks like a rebase/merge conflict mess-up:
These methods were deleted here: https://github.com/brave/brave-core/commit/41ee07f1c9dc2a5209c9c5914aee1f4ea812aba7#diff-065c5e4a04a59cd0bfe79f0ae346447a

Reply:
woops, thanks for the catch. Should be removed now.